docs/dotprompt.md

```diff
@@ -180,9 +190,9 @@ import { promptRef } from "@genkit-ai/dotprompt";
 
 const myPrompt = promptRef("myPrompt");
 
-const result = await myPrompt.generate<typeof MySchema>({...});
+const result = await myPrompt.generate<typeof MyInputSchema, typeof MyOutputSchema>({...});
```

Comment:
Looks like the first generic parameter of `.generate` is model custom options schema.
You don't need to define `MyInputSchema` for this example.

```suggestion
const result = await myPrompt.generate<any, typeof MyOutputSchema>({...});
```

Reply:
Gotcha thanks, is it possible to add strongly typed inputs too? But I guess that's more than a doc edit. What would be good example custom options to add there instead of any please?

Reply:
I'm wondering if it worth updating these anymore. Prompts have undergone pretty significant changes in the next branch for the next release (very soon): https://github.com/firebase/genkit/blob/64cd023d7c645857fe21c4c5404d0e58a1e27f1e/js/ai/src/prompt.ts#L80